File: Array | Dictionary | Set/is_subsequence/py/is_subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        indx = 0

        for i in range(len(t)):
            if indx < len(s) and s[indx] == t[i]:
                indx = indx+1
        
        if indx == len(s):
            return True
        
        return False

File: Array | Dictionary | Set/is_subsequence/ts/is_subsequence.ts
function isSubsequence(s: string, t: string): boolean {
    let indx = 0;

    for ( let i = 0 ; i < t.length; i++ ) {
        if ( indx < s.length && s[indx] == t[i] ) {
            indx = indx+1;
        }
    }

    if (indx == s.length) {
        return true;
    }

    return false;
};